google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/BigtableCloudMonitoringExporter.java

```diff
@@ -49,10 +51,17 @@
  * <p>The exporter will look for all bigtable owned metrics under bigtable.googleapis.com
  * instrumentation scope and upload it via the Google Cloud Monitoring API.
  */
-final class BigtableCloudMonitoringExporter implements MetricExporter {
+@InternalApi
+public final class BigtableCloudMonitoringExporter implements MetricExporter {
 
   private static final Logger logger =
       Logger.getLogger(BigtableCloudMonitoringExporter.class.getName());
+
+  private static final String MONITORING_ENDPOINT =
```

Comment:
please add a comment as to what this is and how its used and who is the intended audience